From 2718ff96f5039569a0fb1b5ffafbb6596e75a53a Mon Sep 17 00:00:00 2001
From: Beverly <beverlyt@google.com>
Date: Mon, 12 Feb 2018 16:45:54 -0500
Subject: [PATCH 129/306] Haptic feedback when ringer set to vibrate

Test: manual
Change-Id: Iaf889f2ee313cc9f16b361928ab8b62a41ef277a
Fixes: 70937425
(cherry picked from commit 636de7511d0ff02f39bba67401b1a89c15126efa)
---
 .../src/com/android/systemui/volume/VolumeDialogImpl.java        | 1 +
 1 file changed, 1 insertion(+)

diff --git a/packages/SystemUI/src/com/android/systemui/volume/VolumeDialogImpl.java b/packages/SystemUI/src/com/android/systemui/volume/VolumeDialogImpl.java
index 3cb37647727..1e8e98ca2e4 100644
--- a/packages/SystemUI/src/com/android/systemui/volume/VolumeDialogImpl.java
+++ b/packages/SystemUI/src/com/android/systemui/volume/VolumeDialogImpl.java
@@ -409,6 +409,7 @@ public class VolumeDialogImpl implements VolumeDialog {
             final boolean hasVibrator = mController.hasVibrator();
             if (mState.ringerModeInternal == AudioManager.RINGER_MODE_NORMAL) {
                 if (hasVibrator) {
+                    mController.vibrate();
                     mController.setRingerMode(AudioManager.RINGER_MODE_VIBRATE, false);
                 } else {
                     mController.setRingerMode(AudioManager.RINGER_MODE_SILENT, false);
-- 
2.17.1

